.github/workflows/rust.yml

```diff
@@ -164,13 +141,10 @@ jobs:
     env:
       LIBCLANG_PATH: 'C:\Program Files (x86)\Microsoft Visual Studio\2019\Enterprise\VC\Tools\Llvm\x64\bin'
     steps:
-      - uses: actions/checkout@v2
-      - uses: actions-rs/toolchain@v1
-        with:
-          profile: minimal
-          toolchain: stable
-          override: true
-      - uses: Swatinem/rust-cache@v1
+      - uses: actions/checkout@v4
+      - uses: dtolnay/rust-toolchain@1.74
```

Comment:
I'd say not to download the toolchain for the current msrv. It's going to be done by xtask and I want the version to be pulled dynamically from the root Cargo.toml.

Reply:
Yea this was only an attempt to fix cargo-msrv